docs/topics/mta-rn-known-issues-6-2-0.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * docs/release_notes/master.adoc
+
+:_content-type: REFERENCE
+[id="rn-known-issues-6-2-0_{context}"]
+= Known issues
+
+MTA version 6.2.0 has the following issues.
+
+* link:https://issues.redhat.com/browse/MTA-823[MTA-823] - Report custom metrics on MTA usage back via OpenShift Telemetry
```

Comment:
@HagayVider1 
This not a known issue. This is a new feature.
Do you have a list of known features from QE?

Reply:
Not yet! But we want to be ahead of the curve